Nataly_Bondarenko/HW5/task3_string.py

```diff
@@ -0,0 +1,9 @@
+# Convert string to a dictionary
+string = "name=Amanda=sssss&age=32&&salary=1500&currency=euro"
+rep = string.replace('=sssss&', ' ')
+rep1 = rep.replace('&&', ' ')
+rep2 = rep1.replace('&', ' ')
```

Comment:
actually good. But you can do the replacement in one line:
rep = string.replace('=sssss&', ' ').replace('&&', ' ').replace('&', ' ')


And naming rep is not good

Reply:
Thanks!
did it in one line
pushed a fix